### ISSUE TITLE
Visual Studio 2017 (15.5.6)

### ISSUE THREAD

**Issue by rlillback** [MicrosoftDocs/visualstudio-docs#533]
As written, VS 2017 (15.5.6) prevents us from adding a Nuget Package to a shared library in a Cross-Platform Xamarin application.  Also, if we select a .NET core library as the PCL does not support Microsoft.Net.Http.  Finally, if we build a blank application (e.g. Android) and add a Legacy Cross Platform Library, we have the following errors:
(1) In DataService.cs, the var response = await client.GetAsync(queryString); is not happy because the Task<HttpResponseMessage> does not contain a definition for 'GetAwaiter' and no extension method accepts Task<HttpResponseMessage> type.
(2) In the core class, the .ConfigureAwait(false) throws a similar error about Task<dynamic> not containing a definition for ConfigureAwait.

Therefore, we cannot use the code as it is written.  What are the updated methods to implement this?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 14465304-3ced-6b36-5054-a85ac11e667e
* Version Independent ID: 9d2d9ca3-8d71-b81c-dbbb-bf4b269db034
* [Content](https://docs.microsoft.com/en-us/visualstudio/cross-platform/build-apps-with-native-ui-using-xamarin-in-visual-studio)
* [Content Source](https://github.com/MicrosoftDocs/visualstudio-docs-pr/blob/live/docs/cross-platform/build-apps-with-native-ui-using-xamarin-in-visual-studio.md)
* Service: unspecified

**Comment by gewarren** [comment]
@rlillback thanks for the feedback. We are aware that this topic is out of date. Please try these topics instead:

https://developer.xamarin.com/guides/android/getting_started/hello,android/
https://developer.xamarin.com/guides/ios/getting_started/hello,_iOS/

**Comment by rvanaken** [comment]
I ran into this issue as well.
This article gets hit a lot because it's linked from the VS start page.
From what I can tell, MS intends to use ".NET Standard Libraries" instead of PCL.  The whole example needs to be rewritten: 
[.NET Standard Class Libraries not PCL](https://developercommunity.visualstudio.com/content/problem/190584/net-standard-class-libraries-not-pcl.html)
It looks like they intend to showcase a more appropriate topic from the VS start page, but it does not look like they authored something up to date.  If there is something up to date in MS docs, it's not yet indexed.
I found this on the Xamarin site: [building-xamarin-forms-apps-net-standard](https://blog.xamarin.com/building-xamarin-forms-apps-net-standard/), but it's from June, so it's ancient history!  ;-)
